src/ir_NEC.cpp

```diff
@@ -145,4 +147,5 @@ bool IRrecv::decodeNEC(decode_results *results, uint16_t offset,
     results->address = reverseBits((data >> 16) & UINT16_MAX, 16);
   return true;
 }
-#endif  // DECODE_NEC || DECODE_SHERWOOD || DECODE_AIWA_RC_T501 || DECODE_SANYO
+#endif  // (DECODE_NEC || DECODE_SHERWOOD || DECODE_AIWA_RC_T501 ||
+        // DECODE_SANYO)
```

Comment:
No change other than broken to multiple lines?

Reply:
Actually, there are brackets around the entire clause which were added, and when I tried to join it just now into a single line, it's >80 chars. So, it stays.